tests/test_utils/api_connexion_utils.py

```diff
@@ -14,26 +14,43 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from contextlib import contextmanager
+
 from airflow.api_connexion.exceptions import EXCEPTIONS_LINK_MAP
 from airflow.www.security import EXISTING_ROLES
 
 
-def create_user(app, *, username, role_name, email=None, permissions=None):
+@contextmanager
+def create_user_scope(app, **kwargs) -> None:
```

Comment:
Same here; `kwargs` should be an explicit argument list. Also the return type is wrong.

```suggestion
def create_user_scope(app, *, username, role_name=None, email=None, permissions=None):
```